app/views/events/_form.html.haml

```diff
@@ -28,12 +28,12 @@
                     .col-md-6#col_eg
                         = form.label :game
                     .col-md-6#col_event_game
-                        = form.select(:game, options_for_select(@game))
+                        = form.select(:game, options_for_select(@game)) 
                 .form-group.row#event_players
                     .col-md-6#col_ep
                         = form.label :users
                     .col-md-6#col_players
-                        = form.select(:users, options_for_select(@users))
+                        = collection_check_boxes(:event, :user_ids, @users, :id, :name)
```

Comment:
![image](https://user-images.githubusercontent.com/34404855/61619607-7709f780-ac6f-11e9-8f3e-7a95d28cfe46.png)

:exclamation: This just looks horrendous. There should be a userename displayed not his id because it doesn't mean anything to anyone really. Moreover, Marchel's idea was to add a drobdown box with users alongside a button to add them to the list.